ui/src/pages/frontsite/Documents/DocumentsDetails/LoanRequestForm/state/actions.js

```diff
@@ -18,24 +15,32 @@ export const requestLoanForDocument = (
       type: IS_LOADING,
     });
     const today = toShortDate(DateTime.local());
+
     try {
-      await loanApi.doRequest(documentPid, sessionManager.user.id, {
-        requestExpireDate: requestEndDate,
-        requestStartDate: today,
-        deliveryMethod: deliveryMethod,
-      });
+      const response = await loanApi.doRequest(
+        documentPid,
+        sessionManager.user.id,
+        {
+          requestExpireDate: requestEndDate,
+          requestStartDate: today,
+          deliveryMethod: deliveryMethod,
+        }
+      );
       dispatch(
         sendSuccessNotification(
           'Success!',
           `You have requested to loan this book.`
```

Comment:
taking into account my comment above, this should be removed too. (maybe again another PR)